### PR TITLE
fix(bounty card): update content, fix tokens display

### LINKED ISSUE #671: [DAO Funds] Value numbers in DAO Funds Graph are displayed partially
**Describe the bug**
US Dollars numbers on vertical graph are not displayed correctly if numbers are greater than 4 digits

**Expected behavior**
Values in Funds graph should be fully displayed.

**Screenshots**
Screenshot 2022-02-15 at 13 35 23

### LINKED ISSUE #672: [Bounty] Fungible tokens ticker is displayed incorrecty in the list of bounties
**Expected behavior**
Fungible tokens ticker should be displayed the same way as it displayed for NEAR.

**Screenshots**
Screenshot 2022-02-15 at 13 49 08

### PR DESCRIPTION
**Pull request by VERZUOL1**

### CODE REVIEW

File: astro_2.0/components/Feed/components/FeedFilter/FeedFilter.tsx [hunk 1]
```diff
@@ -10,8 +10,8 @@ import React, {
 import { PAGE_LAYOUT_ID } from 'constants/common';
 
 import { Icon } from 'components/Icon';
-import { RadioGroup } from 'astro_2.0/components/inputs/radio/RadioGroup';
-import { Radio, RadioProps } from 'astro_2.0/components/inputs/radio/Radio';
+import { RadioGroup } from 'astro_2.0/components/inputs/Radio/RadioGroup';
```
Comment (AlexeyPonomarenko) [inline astro_2.0/components/Feed/components/FeedFilter/FeedFilter.tsx]: I consider there is something wrong with the path. Currently radio is starting from the lowercase letter and I believe it's something that should be preserved

File: fix(bounty_card)__update_content,_fix_tokens_display.patch [hunk 2]
```diff
@@ -0,0 +1,791 @@
+Index: astro_2.0/features/Proposals/components/ProposalStatusFilter/ProposalStatusFilter.tsx
```
Comment (AlexeyPonomarenko) [inline fix(bounty_card)__update_content,_fix_tokens_display.patch]: Should this file be here?

File: astro_2.0/components/inputs/radio/Radio/Radio.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import cn from 'classnames';
 
-import { useRadioContext } from './hooks';
+import { useRadioContext } from 'astro_2.0/components/inputs/radio/Radio/hooks';
```
Comment (AlexeyPonomarenko) [inline astro_2.0/components/inputs/radio/Radio/Radio.tsx]: Can we restore previous import?)

Comment (VERZUOL1) [reply astro_2.0/components/inputs/radio/Radio/Radio.tsx]: done

File: fix(bounty_card)__update_content,_fix_tokens_display.patch (hunk 2 above)
Comment (VERZUOL1) [reply fix(bounty_card)__update_content,_fix_tokens_display.patch]: fixed

**Review by AlexeyPonomarenko — APPROVED** [review]

**Pull request merged**